custom_test.py

```
@@ -8,7 +8,23 @@
+import os
+sys.path.append(os.path.join(os.path.dirname(__file__), 'demos', 'test_files'))
+
+try:
+    from test_utils import truncate_text, validate_qadi_result
+except ImportError:
+    # Fallback implementations if test_utils is not available
+    def truncate_text(text: str, max_length: int = 100) -> str:
+        """Fallback text truncation."""
+        return text[:max_length] + "..." if len(text) > max_length else text
+    
+    def validate_qadi_result(result) -> bool:
```

Ruff (0.11.9)

Missing type annotation for function argument `result`

(ANN001)